bitrise.yml

```diff
@@ -0,0 +1,17 @@
+format_version: "11"
+default_step_lib_source: https://github.com/bitrise-io/bitrise-steplib.git
+
+workflows:
```

Comment:
Do not forget to add a `sample` workflow, with the simplest use-case, that can be run by anyone easily.

Reply:
Whoops! Added back in c15947d